modules/nixos/nix-build-distributor.nix

```diff
@@ -7,17 +7,19 @@
 
   nix.distributedBuilds = true;
   nix.buildMachines = [
-    # macos-01 - system integrity protection enabled
-    # currently broken and needs to be reinstalled
+    # macos-01
+    # - m1 cpu
+    # - system integrity protection disabled
     {
       hostName = "167.235.13.208";
       sshUser = "builder";
       protocol = "ssh-ng";
-      system = "aarch64-darwin";
-      maxJobs = 0;
+      system = "x86_64-darwin";
```

Comment:
if this is an m1 cpu, does it work with x86_64 architecture?

Reply:
it uses rosetta 2 to run the foreign binaries. which has a side-effect of populating a binary cache under `/var/db/oah`. which we can only clean having system protection disabled. which we now have for two of the m1 machines :-)